### PR TITLE
Feature/se 45

### LINKED ISSUE #45: Create front-end for the Login
image

### PR DESCRIPTION
**Pull request by julix14**
closes SE2-WS23/documentation#45

image

### CODE REVIEW

File: src/pages/Login/LoginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+      spacing={4}
+      justifyContent="center"
+      alignItems="center"
+      style={{
+        backgroundColor: "#1975D2",
+        color: "#fff",
+        height: "100vh",
+        width: "100vw",
+        margin: 0,
+      }}>
```
Comment (Chrissydrx) [inline src/pages/Login/LoginPage.jsx]: Please use the sx tag for styling in MUI

File: src/pages/Login/LoginPage.jsx [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+  const handleLogin = () => {
+    console.log("Login clicked");
+    console.log("Email: " + email);
+    console.log("Password: " + password);
+  };
+
+  const handleEmailChange = (event) => {
+    setEmail(event.target.value);
+  };
```
Comment (Chrissydrx) [inline src/pages/Login/LoginPage.jsx]: Please add a check if the provided email is a valid email

**Review by Chrissydrx — CHANGES_REQUESTED** [review]

**Comment by Chrissydrx** [comment]
I am not supposed to do design feedback, right?

**Comment by julix14** [comment]
> I am not supposed to do design feedback, right?

Don't think so

**Review by Chrissydrx — APPROVED** [review]

**Pull request merged**